source/manual/deploying-terraform.html.md

```diff
@@ -9,77 +9,80 @@ review_in: 3 months
 
-### 2. Install `gds-cli`
+## Deploying on Jenkins
 
-See [instructions for installing `gds-cli`](/manual/access-aws-console.html).
+> **This is not the recommended approach.**
+>
+> It's slightly easier to start with, but requires you to push your changes to GitHub and you'll quickly want to switch to deploying locally (see the next section) to avoid pasting secrets over and again!
 
-### 3. Create a read-only GitHub personal access token
+### 1. Generate an AWS token and key
 
-Create a [Github personal access token](https://github.com/settings/tokens) with the `read:org` scope.
+First make sure you have [`gds-cli`](/manual/access-aws-console.html) installed.
 
-  * Follow the link above and press the `Generate a new token` button near the
-    top of the page.
-  * Enter a suitable description for the token under `Note`, for example `Token
-    for Jenkins Terraform deployments`.
-  * Under `Select scopes`, tick the `read:org` box **only**.
-  * Press `Generate token` at the bottom of the form.
-  * Please take care to store and handle the token securely. It allows access
-    to Jenkins. If you accidentally share your token,
-    [revoke it immediately](https://github.com/settings/tokens) and follow the
-    [instructions for reporting a potential data security incident][security-incidents].
+👉 [Instructions for installing `gds-cli`](/manual/access-aws-console.html)
 
-[security-incidents]: https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/working-at-the-white-chapel-building/security/security-incidents
+Run the following to generate a token and key.
 
-## Plan and deploy a Terraform project via Jenkins using `deploy.rb`
+```
+gds aws <your-role e.g. govuk-integration-poweruser> -e
+```
 
-The Ruby script [`tools/deploy.rb`][deploy-rb] in the `govuk-aws` repository takes care of requesting temporary AWS credentials with an assumed role and queuing the deployment Jenkins job.
+### 2. Run the Jenkins job
+
+**Always `plan` first, check that the output is what you expect, then `apply`.**
 
-To follow these instructions you will need:
+Manually run the Jenkins job, pasting the token and key from the previous step.
 
-* The name of the Terraform project which you want to deploy, for example `app-backend`.
-* The stackname of the Terraform project which you want to deploy. Usually, this is `blue` for `app-` projects and `govuk` for `infra-` projects.
-* Your GitHub username.
-* Your GitHub personal access token (see setup instruction above).
-* The name of your AWS role (`govuk-powerusers` or `govuk-adminstrators`).
-* A local copy of the `govuk-aws` Git repo, which contains the `deploy.rb` script. The example command assumes that your local repo is in the directory `~/govuk/govuk-aws`.
+👉 [Deploy terraform directly from Jenkins][ci-deploy-jenkins]
 
-**Always `plan` first, check that the output is what you expect, then `apply`.**
 
-1. To start the Terraform plan job, run `deploy.rb` like this.
+## Deploying locally
```

Comment:
This still deploys on Jenkins, it's just less effort for users. I can't think how to word this better though.

Reply:
To me, "deploying locally" means "running `terraform apply` from your laptop", which we don't do because audit trails are good.

Reply:
Hmm...I see what you mean. The titles are pretty superficial, but I guess the function of `deploy.rb` is somewhat vague (it doesn't actually deploy anything by itself), so the best title is probably something equally mysterious...updated!